src/Traits/Comparable.php

```diff
@@ -4,6 +4,9 @@
 
 use PHLAK\SemVer\Version;
 
+/**
+ * Trait for comparing two version objects.
+ */
```

Comment:
I'd prefer to omit unnecessary doc blocks and comments.

```suggestion
```

Reply:
I think it’s bad practice not to PHPDoc (public) classes, traits, interfaces, functions. You have to decide. 🙃